app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/apiservices/PokemonService.kt

```diff
@@ -0,0 +1,10 @@
+package au.com.gridstone.trainingkotlin.apiservices
+
+import au.com.gridstone.trainingkotlin.data.PokemonResults
+import retrofit2.Response
+import retrofit2.http.*
```

Comment:
According to the style guide, [avoid wildcard imports](https://github.com/Gridstone/KotlinStyleGuide#imports)